tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -1035,6 +1035,41 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         jqUnit.assertUndefined("Conditional component should not have been constructed on startup", thatWithout.conditionalComponent);
     });
 
+    // The model relay between the parent component and a conditional component stops the contruction of the entire component
+    fluid.defaults("fluid.tests.fluid6603", {
+        gradeNames: "fluid.modelComponent",
+        model: {
+            constructComp: true,
+            relayValue: null
+        },
+        dynamicComponents: {
+            conditionalComponent: {
+                source: "{that}.model.constructComp",
+                type: "fluid.modelComponent",
+                options: {
+                    model: {
+                        relayValue: true
+                    },
+                    modelRelay: {
+                        connectRelayValue: {
+                            source: "relayValue",
+                            target: "{fluid6603}.model.relayValue",
+                            // forward: "liveOnly"
+                            excludeSource: "init"
```

Comment:
This needs to say
forward: {
    excludeSource: "init"
}

Reply:
I adjusted the `forward` syntax to the suggested, the same issue still persists.